<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidgets</class>
 <widget class="QMainWindow" name="ListWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageQListWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>428</width>
          <height>649</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget操作</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>111</width>
           <height>361</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QToolButton" name="toolButtonListInit">
            <property name="text">
             <string>初始化列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonListClear">
            <property name="text">
             <string>清除列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonListInsert">
            <property name="text">
             <string>插入列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonListAppend">
            <property name="text">
             <string>添加列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonListDelete">
            <property name="text">
             <string>删除当前项</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="pageQTreeWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>428</width>
          <height>649</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget操作</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>节点</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>节点类型</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>图片文件</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
            <item>
             <property name="text">
              <string>分组节点</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>图片节点</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>分组2</string>
             </property>
             <item>
              <property name="text">
               <string>图片2</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </item>
            </item>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageQTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>428</width>
          <height>649</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTableWidget操作</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabQListWidget">
        <attribute name="title">
         <string>QListWidget</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>当前项变化</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEditCutItemText"/>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBoxCanEdit">
              <property name="text">
               <string>可编辑</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QToolButton" name="toolButtonSelItem">
              <property name="text">
               <string>选择项</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="toolButtonSelAll">
              <property name="text">
               <string>全选</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QToolButton" name="toolButtonSelNone">
              <property name="text">
               <string>取消全选</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QToolButton" name="toolButtonSelInvs">
              <property name="text">
               <string>反选</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabQTreeWidget">
        <attribute name="title">
         <string>QTreeWidget</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>514</width>
              <height>687</height>
             </rect>
            </property>
            <widget class="QLabel" name="labelTreeWiget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>591</width>
               <height>561</height>
              </rect>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabQTableWidget">
        <attribute name="title">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionListInit">
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actionListClear">
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actionListInsert">
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="actionListAppend">
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="actionListDelete">
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="actionSelAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelNone">
   <property name="text">
    <string>取消全选</string>
   </property>
   <property name="toolTip">
    <string>取消全选</string>
   </property>
  </action>
  <action name="actionSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSelPopMenu">
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionZoomRealSize">
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actionZoomFitW">
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
